createBookFromReadme.sh

```diff
@@ -18,15 +18,33 @@ function cleanupBeforeStarting(){
 
 # Splits the Readme.md file based on the header in markdown and creates chapters
 # Note:
-#   Get gcsplit via homebrew on mac: brew install coreutils
+#   Get csplit via homebrew on mac: brew install coreutils
```

Comment:
This is again wrong. On macOS you are using the gcsplit still on line 32 in this file. So saying csplit is being installed here is wrong.

You might want to write this:

#   For macOS: Get gcsplit via homebrew, cmd: brew install coreutils
#   For Linus: Get csplit via distro's package manager